backend/typescript/prisma/schema.prisma

```diff
@@ -0,0 +1,121 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Post {
+  id        Int      @id @default(autoincrement())
+  createdAt DateTime @default(now())
```

Comment:
I think we should change the name to created_at instead because querying using SQL will be easier and more consistent